<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionDisplayMessage</class>
 <widget class="QWidget" name="ActionDisplayMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblMessageIDs">
       <property name="text">
        <string>Message IDs:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DynamicList" name="messageIDs" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DynamicList</class>
   <extends>QWidget</extends>
   <header>DynamicList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
